cfgov/v1/wagtail_hooks.py

```diff
@@ -365,21 +366,39 @@ def hide_snippets_menu_item(request, menu_items):
                      if item.url != reverse('wagtailsnippets:index')]
 
 
-# Override list of allowed tags in a RichTextField
-@hooks.register('construct_whitelister_element_rules')
-def whitelister_element_rules():
-    allow_html_class = attribute_rule({
-        'class': True,
-        'itemprop': True,
-        'itemscope': True,
-        'itemtype': True,
-    })
+if wagtail.VERSION <= (1, 14):
```

Comment:
Since it's a `<=` comparison, this can maybe be slightly clearer by checking for Wagtail 1.13.

```suggestion
if wagtail.VERSION <= (1, 13):
```